R/format_geoparquet.R

```diff
@@ -0,0 +1,16 @@
+format_geoparquet <- tar_format(
+  read = function(path) {
+    arrow::read_parquet(path) |>
+      dplyr::mutate(dplyr::across(wk::is_handleable, wk::as_wkb))
+  },
+  write = function(object, path) {
+    arrow::write_parquet(object, path)
```

Comment:
I needed to convert the SpatVector -> data.frame for this to work (EDIT: I left out `geom="WKT"` to preserve geometry). 

```suggestion
    arrow::write_parquet(as.data.frame(object, geom="WKT"), path)
```